framework/src/packets/icmp/v6/ndp/mod.rs

```diff
@@ -1,6 +1,6 @@
 mod neighbor_advert;
 mod neighbor_solicit;
-mod options;
+pub mod options;
```

Comment:
probably don't need to pub this, and just export through to options as we've done in other mods, right @drunkirishcoder ?

Reply:
fixed per your PR